### PR TITLE
Add tomcat into thirdparty_containers

### PR DESCRIPTION
**Pull request by chandrams**
Included dockerfile and scripts to run tomcat tests for 8.5.24 version.  

Issue #281 

Signed-off-by: Chandrakala <chandra-ms@in.ibm.com>

### CODE REVIEW

File: thirdparty_containers/tomcat/build.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<project name="tomcat-test" default="build" basedir=".">
+	<taskdef resource="net/sf/antcontrib/antlib.xml" />
+	<description>
+		Build tomcat-test Docker image
+	</description>
+
+	<!-- set properties for this build -->
+	<property name="DEST" value="${BUILD_ROOT}/thirdparty_containers/tomcat-test" />
+	<property name="dist" location="${DEST}/${JAVA_VERSION}" />
```
Comment (sophia-guo) [inline thirdparty_containers/tomcat/build.xml]: With new TestKitGen we don't need the "dist" for different JAVA_VERSION any more. 'DEST' is enough.

**Comment by Mesbah-Alam** [comment]
Is there a link to a personal build where you ran this change?

**Comment by chandrams** [comment]
Couldn't find the older runs, submitted one today - https://ci.adoptopenjdk.net/view/work%20in%20progress/job/test_personal/381/

File: thirdparty_containers/tomcat/build.xml (hunk 1 above)
Comment (smlambert) [reply thirdparty_containers/tomcat/build.xml]: @sophia-guo - so what specific change are you asking for @chandrams to make?  Are you asking to remove line 10, and any references elsewhere to dist?

Comment (sophia-guo) [reply thirdparty_containers/tomcat/build.xml]: Yes, line 10 can be removed. Correspondingly line 15 should be changed

File: thirdparty_containers/tomcat/build.xml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	<property name="src" location="." />
+	<property name="jvm_version" location="${JVM_VERSION}" />
+
+	<target name="init">
+		<mkdir dir="${dist}"/>
```
Comment (sophia-guo) [inline thirdparty_containers/tomcat/build.xml]: Change to ```<mkdir dir="${DEST}"/>```

Comment (chandrams) [reply thirdparty_containers/tomcat/build.xml]: I have made these changes, please review.

**Pull request merged**